jsonnet/telemeter/client/telemeter-client.libsonnet

```diff
@@ -0,0 +1,11 @@
+(import 'kubernetes/kubernetes.libsonnet') + {
+  _config+:: {
+    namespace: 'default',
+
+    telemeterClientSelector: 'job="telemeter-client"',
+
+    jobs: {
+      TelemeterClient: $._config.telemeterClientSelector,
```

Comment:
TelemeterClient -> telemeterClient

Reply:
I was following the convention in https://github.com/coreos/prometheus-operator/blob/master/contrib/kube-prometheus/jsonnet/kube-prometheus/kube-prometheus.libsonnet#L38-L46

should they all be lowercase?

Reply:
that's strange, I don't know why we ever started doing that, we should change that in the kubernetes mixin as well

Reply:
good catch!